apps/home/stylesheets/hero.styl

```diff
@@ -1,38 +1,54 @@
+bp = 1024px
+
 .HomeHero
   &__focus
     position relative
     padding 0 2em
     font-size font-size-h2
     margin 2em auto
 
-  &__demo-block
+  &__content
     display flex
-    justify-content flex-start
-    width block_width
-
-    > .Block
-      margin 0 auto
+    width 100%
+    flex-direction row
+    justify-content center
+    align-items flex-start
+    flex 1
+    padding 6em 1em
+    +below(bp)
+      flex-direction column
+      flex-flow column-reverse
 
-      // Hide default overlay buttons (Source / Connect)
-      .Block__inner__overlay
-        display none
-      // Enable border
-      .Block__inner
-        border-color colors-gray-border
-      // Hide title & timestamp, display attribution
-      .Block__caption__meta
-        display block
-      .Block__caption__title
-      .Block__caption__meta__timestamp
-        display none
-      // Relative caption
-      .Block__caption
-        position relative
+  &__copy
+    width 25%
+    margin-right 2em
+    display flex
+    flex-direction column
+    align-items center
+    align-self center
+    +below(bp)
+      width 100%
+      padding 0 1em
+      margin-right 0
 
-    // Enabled when hovering faux 'Connect' button
-    &--hover
-      opacity 0.5
+  &__cta
+    width 100%
+    margin-top 2em
+    > .Button--app
+      display none
+    +below(bp)
+      text-align center
+      > .Button--app
+        display inline-block
+        margin 1em
 
+  &__image
+    width 60%
+    align-self center
+    +below(bp)
```

Comment:
It is perhaps personal preference, but I'd rather the response styles be nested under a single media query, rather than littered around. I never understood this style though.

Reply:
I can do that, I just always found that way a bit harder to parse.